WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -243,11 +241,21 @@ private extension HubMenuViewModel {
          Updating the bottom UIEdgeInset to a non-zero value seems to be enough to trigger the UI layout refresh we need.
          Ref: gh-13785
          */
-        if mainTabBarController.tabBar.isHidden {
-            let bottomInset = CGFloat.leastNonzeroMagnitude
-            mainTabBarController.additionalSafeAreaInsets = UIEdgeInsets(top: 0, left: 0, bottom: bottomInset, right: 0)
+        if isPointOfSaleActive {
+            UIView.animate(withDuration: 0.5) {
+                mainTabBarController.tabBar.alpha = 0
+            } completion: { _ in
+                mainTabBarController.tabBar.isHidden = isPointOfSaleActive
```

Comment:
Since we hide the tab bar only after the animation completes, the loading view suddenly moves down:

https://github.com/user-attachments/assets/317193c3-c366-4862-88ec-70c9ce056012

I tried adding `.ignoresSafeArea()` to `PointOfSaleLoadingView` which seems to do the trick and immediately put the loading view in a position that ignores the existence of tab bar.

Reply:
That's neat, it also works on my end and the little jump while the LoadingView is in-flight disappears 💯 , updated on https://github.com/woocommerce/woocommerce-ios/pull/13905/commits/67ef3461324ed05d33a7d923e014f82f48954178